internal/options/options.go

```diff
@@ -247,6 +247,14 @@ func Init() (err error, errCb func()) {
 		}
 	}
 
+	yamlOptions, err := Yaml()
+	if err != nil {
+		return nil, nil
```

Comment:
Are there no other types of errors you would want to return like if there were parsing errors etc?

Reply:
Ah whoops, that's a typo - we should return the error